app/components/stanford_only_popover_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class StanfordOnlyPopoverComponent < StanfordOnlyIconComponent
```

Comment:
I don't think these should inherit as this is not a natural inheritance relationship.  The popover component can contain a StanfordOnlyIconComponent, instead right?

Reply:
Then the StanfordOnlyIconComponent would need to take a variable because the aria variables are different if it is an svg vs a button.

Reply:
How about always making the SVG `aria-hidden="true"` and wrap it with a `span aria-label="stanford only"` when not used in a button.